frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -59,3 +60,23 @@ export const getOCSVersion = (items: FirehoseResult): string => {
   );
   return _.get(operator, 'status.installedCSV');
 };
+
+export const getSCAvailablePVs = (sc: string) =>
+  k8sList(PersistentVolumeModel)
+    .catch(() => [])
```

Comment:
Shouldn't catch be after then?
```suggestion
    .catch(() => [])
```

Reply:
Not necessary